js/pages/data-sources/data-sources.js

```diff
@@ -104,8 +104,8 @@ define([
 			];
 
 			this.sources = ko.computed(() => sharedState.sources().filter(function (s) {
-                return s.hasResults && s.hasVocabulary;
-            }));
+				return s.hasResults && s.hasVocabulary && authApi.isPermittedViewDataSourceReport(s.sourceKey);
```

Comment:
No sources would be available when authentication is turned off

Reply:
See above